optional_plugins/mail/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include VERSION README.rst
```

Comment:
Hi @harvey0100, still new line missing here. :wink:

Reply:
I'll get there one day haha, IDE is now set to save all files with a new line :D - sorted now